### PR TITLE
chore: Add README, LICENSE and a simple example

### PR DESCRIPTION
**Pull request by pingyu**
Preview [here](https://github.com/pingyu/tracing-active-tree/tree/readme).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+```
+
+2. Create spans that you wish to display in the calling trees:
+
+```rust
+#[instrument(fields(answer = 43))]
+async fn foo() {
+    bar().await;
+}
+
```
Comment (HuSharp) [inline README.md]: Is it better to indent after an ordered list?
```suggestion
1. Import the library:

	```rust
	use tracing_active_tree::layer::{self, CurrentStacksLayer};
	```

2. Create spans that you wish to display in the calling trees:

	```rust
	#[instrument(fields(answer = 43))]
	async fn foo() {
	    bar().await;
	}

```

Comment (pingyu) [reply README.md]: Fixed, PTAL~

p.s. other warnings from [`markdownlint`](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) were also fixed.

**Review by HuSharp — APPROVED** [review]
LGTM~

**Review by YuJuncen — APPROVED** [review]
LGTM, thank you!

**Pull request merged**